src/App.tsx

```diff
@@ -17,7 +21,24 @@ export const AuthContext = createContext<Partial<AuthContext>>({})
 type WebsocketContext = InstanceType<typeof Client> | null
 export const WebsocketContext = createContext<WebsocketContext>(null)
 
+// options for different color modes
+const modes = [
+  'light',
+  'dark',
+]
+
+// merge the color mode with the base theme
+// to create a new theme object
+const getTheme = mode =>
+  merge({}, baseTheme, {
+    colors: get(baseTheme.colors.modes, mode, baseTheme.colors),
+  })
+
 const App: React.FC = () => {
+  // state for changing modes dynamically
+  const [mode, setMode] = useState(modes[0])
```

Comment:
Q:
Is there an expectation that any modes will be set besides `'light'` and `'dark'`?
If no, then a simpler way to go about this would be something more like:
`const [isDarkMode, setIsDarkMode] = useState(false)`

That would simplify the logic around line 82

Reply:
I should have mentioned this in the PR, but really, this bit was just to prove that it worked. This code, which proves that a button could be pushed to "toggle" a color mode, should live somewhere else in the app—like a settings view or something. Therefore, it is at that time, I will take your feedback into consideration! ;)

Reply:
For now, no. I don't see the need for any other color modes. Will update.